requirements/test.txt

```diff
@@ -0,0 +1,4 @@
+-r common.txt
```

Comment:
Ok, so the plan is to have this separate testing file, and then leave regular `requirements.txt` in place as an alternative to `pip install` yes?

Reply:
Ah ok, i see your comments  in #93